lib/capybara/selector/definition/button.rb

```diff
@@ -40,6 +40,8 @@
     builder(xpath).add_attribute_conditions(name: val)
   end
 
+  node_filter(:focused) { |node, value| (node.native == node.session.active_element.native) == value }
```

Comment:
@twalpole this is a duplicate of the `_field` filter set.

In practice, _any_ element with a `[tabindex]` attribute can have focus. Is there a way to declare a global filter?

Reply:
global filters are defined in SelectorQuery - see `matches_id_filter?`, ``matches_class_filter?`, etc

Reply:
Although it might be better to just add it to a fieldset and them import it into the different selectors.